apps/volontulo/views/pages.py

```diff
@@ -12,19 +14,24 @@
 from apps.volontulo.models import Page
 
 
-# pylint: disable=too-many-ancestors;missing-docstring
+# pylint: disable=too-many-ancestors,missing-docstring
 class PageList(ListView):
     model = Page
     template_name = 'pages/page_list.html'
 
+    @method_decorator(user_passes_test(
```

Comment:
You have the same `lambda` in 4 places. Please create named function somewhere and pass it here.

Second thing here is that pylint is failing on Your 8 spaces indentation.